### PR TITLE
add example: add new model to data labeler

### PR DESCRIPTION
**Pull request by AnhTruong**
address the issue https://github.com/capitalone/DataProfiler/issues/12

### CODE REVIEW

**Review by lettergram — COMMENTED** [review]
I believe there needs to be a more sizable explanation on what's going on here. To me, I just see a wall of text. While I see a model is designed, I would need further assistance to get it to work. I'm not really sure how it fits in the pipeline etc.

Recommendations:

* Reduce the actual code portions where possible
* Expand comment descriptions
* Possibly add some infographics or text explaining the pipeline `step 1` -> `step 2` -> etc

File: examples/add_new_model_to_data_labeler.ipynb [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+    "value_label_df = value_label_df.astype(str)\n",
+    "labels = value_label_df[1].unique().tolist()\n",
+    "\n",
+    "# build new data labeler with the new lstm model\n",
+    "data_labeler = dp.DataLabeler(labeler_type='structured', trainable=True)\n",
+    "model = CharacterLevelLstmModel(labels + ['UNKNOWN']) # extend the labels with the default label 'UNKNOW'\n",
+    "data_labeler.set_model(model)\n",
+    "\n",
+    "# set default label (one of the column names) to the data labeler pipeline\n",
+    "params = {'default_label': 'comment'}\n",
```
Comment (gme5078) [inline examples/add_new_model_to_data_labeler.ipynb:340]: Why do we have this and the above part with adding the UNKNOWN? Only one of these should be needed right?

Comment (gme5078) [reply examples/add_new_model_to_data_labeler.ipynb:340]: And can you please add a conclusion at the end?

File: examples/add_new_model_to_data_labeler.ipynb [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+   "source": [
+    "# get labels from the given dataset\n",
+    "value_label_df = data_train.reset_index(drop=True).melt()\n",
+    "value_label_df.columns = [1, 0]  # labels=1, values=0 in that order\n",
+    "value_label_df = value_label_df.astype(str)\n",
+    "labels = value_label_df[1].unique().tolist()\n",
+    "\n",
+    "# build new data labeler with the new lstm model\n",
+    "data_labeler = dp.DataLabeler(labeler_type='structured', trainable=True)\n",
+    "model = CharacterLevelLstmModel(labels + ['UNKNOWN']) # extend the labels with the default label 'UNKNOW'\n",
```
Comment (gme5078) [inline examples/add_new_model_to_data_labeler.ipynb:336]: Typo with "UNKNOW"

**Review by gme5078 — CHANGES_REQUESTED** [review]

**Review by gme5078 — APPROVED** [review]

**Review by lettergram — APPROVED** [review]

**Pull request merged**